gladiator.js

```diff
@@ -0,0 +1,10 @@
+class gladiator(name, weapon){
+  this.name: name;
+  this.weapon: weapon;
```

Comment:
The assignment of these properties must happen inside of a constructor, like you did with the `Arena` class.